src/app/zones/views/ZoneCreateView.vue

```diff
@@ -41,17 +41,45 @@
         <KCard>
           <template #body>
             <div class="form">
-              <div>
-                <h1 class="form-title">
-                  {{ t('zones.routes.create.title') }}
-                </h1>
+              <div class="form-header">
+                <div>
+                  <h1 class="form-title">
+                    <span class="text-gradient">{{ t('zones.form.title') }}</span>
+                  </h1>
+
+                  <p v-if="t('zones.form.description') !== ' '">
+                    {{ t('zones.form.description') }}
+                  </p>
+                </div>
+
+                <div
+                  v-if="tm('zones.form.facts').length > 0"
+                  class="fact-list"
+                >
+                  <div
+                    v-for="(fact, index) in tm('zones.form.facts')"
+                    :key="index"
+                    class="fact-list__item"
+                  >
+                    <KIcon
+                      icon="check"
+                      :color="KUI_COLOR_TEXT_SUCCESS"
+                    />
+
+                    {{ fact }}
+                  </div>
+                </div>
```

Comment:
Bit of a nit but should we use a `ul/li` here instead of `div`s?

Reply:
Changed this.